microk8s-resources/actions/fluentd/fluentd-es-ds.yaml

```diff
@@ -110,7 +108,7 @@ spec:
             path: /var/log
         - name: varlibdockercontainers
           hostPath:
-            path: /var/snap/microk8s/common/var/lib/containerd
+            path: /var/lib/docker/containers
```

Comment:
@balchua this feels wrong.

Reply:
Oh im sorry about that. I'll work on this.